app/views/admin/_admin_options.html.erb

```diff
@@ -1,16 +1,42 @@
 <!-- BEGIN admin options -->
-  <div class="admin">
-	<h3 class="landmark heading">Admin Actions</h3>
+<div class="admin">
+  <% item_type_name = item.class == ExternalWork ? "External Work" :
```

Comment:
We've still been using `ts()` in our regular view files, so until we talk this over and formally decide otherwise, I'm sticking with that.

If I'm making External Work translatable, what would be the proper way to make it so `item_class_name` also had translatable strings for Work and Bookmark? Something like this?

```
<% if item.class == ExternalWork %>
  <% item_type_name = ts("External Work") %>
<% elsif item.class == Bookmark %>
  <% item_type_name = ts("Bookmark") %>
<% else %>
  <% item_type_name = ts("Work") %>
<% end %>
```